cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -133,6 +133,20 @@ func UpdateToUEFICompatible(workflow *daisy.Workflow) {
 				disk.Disk.GuestOsFeatures = daisy.CombineGuestOSFeatures(disk.Disk.GuestOsFeatures, "UEFI_COMPATIBLE")
 			}
 		}
+		if step.CreateDisksAlpha != nil {
```

Comment:
Is this a new type of step? In similar changes, for images and instances, we've kept the existing step but added support in it for different release tracks

Reply:
I chose this way to proactively determine whether alpha-version API needs to be called. Sometimes a single step will try to create 3 disks, but only one of them rely on alpha feature.
Anyways I hope it is migrated to GA when we are ready to release, thus we don;t need these code anymore. At least beta.